### PR TITLE
Add security notice to joomla/crypt

### PR DESCRIPTION
**Pull request by paragonie-scott**
Depends on #8406 for its advice to hold any validity.

### CODE REVIEW

**Comment by paragonie-scott** [comment]
Not sure why this failed (most likely a runtime error with Travis CI).

File: libraries/joomla/crypt/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+```
+
+## Avoid these Ciphers if Possible
+
+* `JcryptCipher3Des`
+* `JcryptCipherBlowfish`
+* `JcryptCipherMcrypt`
+* `JcryptCipherRijndael256`
+
+All of these ciphers are vulnerable to something called a [chosen-ciphertext attack](https://en.wikipedia.org/wiki/Chosen-ciphertext_attack). The only provable way to prevent chosen-ciphertext attacks is to [use uthenticated encryption](https://paragonie.com/blog/2015/05/using-encryption-and-authentication-correctly), preferrably in an [Encrypt-then-MAC construction](http://www.thoughtcrime.org/blog/the-cryptographic-doom-principle/).
```
Comment (tarcieri) [inline libraries/joomla/crypt/README.md]: `uthenticated`? :stuck_out_tongue_winking_eye:

Comment (paragonie-scott) [inline libraries/joomla/crypt/README.md]: Whoops. :)

File: libraries/joomla/crypt/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Important Security Information
+
+If you're going to use JCrypt in any of your plugins, make *sure* you use **JCryptCipherCrypt**; it's the only one that's cryptographically secure. (It's [version 1.1 of Defuse Security's encryption library](https://github.com/defuse/php-encryption)).
+
+```php
+$cipher = new JCryptCipherCrypto();
+$key = $cipher->generateKey(); // Store this for long-term use
+$ciphertext = $cipher->encrypt($message, $key);
```
Comment (roland-d) [inline libraries/joomla/crypt/README.md]: @paragonie-scott The $message is not defined. Do you want to add that to the readme?

Comment (paragonie-scott) [inline libraries/joomla/crypt/README.md]: Sure.

**Comment by paragonie-scott** [comment]
Does anyone object to adding this README?

**Comment by Bakual** [comment]
No objections from me. Setting RTC from review.

**Comment by Bakual** [comment]
@joomla-cms-bot Behave! :smile:

**Comment by Kubik-Rubik** [comment]
Thank you @paragonie-scott! Merged.

**Pull request merged**